app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -624,6 +625,11 @@ class BrowserFragment :
         }
     }
 
+    override fun onStop() {
```

Comment:
This fix is not ok . Will cause a crash 
 Caused by: kotlin.UninitializedPropertyAccessException: lateinit property tabsPopup has not been initialized
        at org.mozilla.focus.fragment.BrowserFragment.onStop(BrowserFragment.kt:630)